src/charm.py

```diff
@@ -177,20 +180,13 @@ def _set_mimir_config(self) -> bool:
             logger.error("Failed to push updated config file: %s", e)
             raise BlockedStatusError(str(e))
 
-    def _set_alerts(self) -> bool:
-        """Create alert rule files for all Mimir consumers.
+    def _set_alerts(self) -> None:
```

Comment:
@rbarry82 say we have 200 (2000?) relations/rule files. Shouldn't we avoid pushing them all over again each time unnecessarily?

Reply:
It really doesn't matter from Mimir's POV. We hash for prom so we don't restart unnecessarily and risk a tsdb rebuild. Mimir doesn't need restarting, so...

Reply:
I meant from pebble iops reliability point of view.